aten/src/ATen/native/cpu/SortingKernel.cpp

```diff
@@ -2,13 +2,131 @@
 #include <ATen/Dispatch.h>
 #include <ATen/Parallel.h>
 #include <ATen/NumericUtils.h>
+#include <ATen/native/TensorIterator.h>
```

Comment:
I wonder if we really need this code to sit inside of /native/cpu as it is not benefiting from AVX support.